src/main/webapp/app/components/util/markdown.service.ts

```diff
@@ -155,6 +155,20 @@ export class ArtemisMarkdown {
         return this.sanitizer.bypassSecurityTrustHtml(sanitized);
     }
 
+    /**
+     * Converts markdown into html, sanitizes it and then declares it as safe to bypass further security.
+     *
+     * @param {string} markdownText the original markdown text
+     * @returns {string} the resulting html as a string
+     */
+    htmlForGuidedTourMarkdown(markdownText: string | null) {
```

Comment:
I originally used the `htmlForMarkdown` pipe, but it sanitized too much and didn't allow, e.g. the attribute `target="_blank"`. Therefore I added a new pipe that should explicitly check the guided tour content input.